csource/common.go

```diff
@@ -198,6 +279,24 @@ static uintptr_t syz_fuseblk_mount(uintptr_t a0, uintptr_t a1, uintptr_t a2, uin
 	return fd;
 }
 
+#ifdef __NR_syz_emit_ethernet
+struct syz_blob {
+	int64_t	length;
+	int64_t	reserved;
+	char	data[0];
+};
+
+static uintptr_t syz_emit_ethernet(uintptr_t a0)
```

Comment:
Documentation/networking/tuntap.txt claims that
"Depending on the type of device chosen the userspace program has to read/write
IP packets (with tun) or ethernet frames (with tap)".
The function is called syz_emit_ethernet, but it uses tun so it emits IP frames (?). If so, it should not be called _ethernet.
On a side note, we probably also want to emit ethernet frames as well. Or do we want to emit only ethernet as lower level proto?